autoload/neomake/makers/ft/go.vim

```diff
@@ -1,7 +1,7 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#go#EnabledMakers() abort
-    return ['go', 'golint', 'govet']
+    return ['go', 'golint', 'govet', 'gometalinter']
```

Comment:
They won't always report compile errors, as some of them will just say "can't load project", others will point out the exact line number - but in all cases, only the go compiler is authoritative on what's a compile error - the others use similar but not exactly the same tools.